app/src/main/java/io/homeassistant/companion/android/notifications/NotificationsActivity.kt

```diff
@@ -0,0 +1,227 @@
+package io.homeassistant.companion.android.notifications
+
+import android.content.ActivityNotFoundException
+import android.content.Intent
+import android.database.Cursor
+import android.net.Uri
+import android.os.Bundle
+import android.view.ActionMode
+import android.view.Menu
+import android.view.MenuItem
+import android.view.View
+import android.widget.AbsListView
+import android.widget.AdapterView
+import android.widget.ListView
+import androidx.appcompat.app.AppCompatActivity
+import androidx.cursoradapter.widget.SimpleCursorAdapter
+import io.homeassistant.companion.android.R
+import io.homeassistant.companion.android.settings.SettingsActivity
+import kotlinx.android.synthetic.main.activity_notifications.*
+
+class NotificationsActivity : AppCompatActivity() {
```

Comment:
Do we want this to be a full activity or just a fragment under the settings?

Reply:
Ha, I don't know, you're the oracle :)

I would prefer to keep settings completely separate from, non-settings on device activities, but it's up to you.